.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_DB: tokite_test
+          POSTGRES_PASSWORD: postgres_password
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: .ruby-version
+        bundler-cache: true
+    - name: Setup DB
+      run: |
+        RAILS_ENV=test bundle exec rails app:tokite:ridgepole:install app:tokite:ridgepole:apply
+    - name: Run tests
+      run: bundle exec rspec
```

Comment:
nit: On CircleCI, RSpec is configured to show some slow tests in logs. How about doing the same in GitHub Actions?

https://github.com/cookpad/tokite/blob/39a543f8fcae9f017e11aa801efdb5439d5e2fc8/.circleci/config.yml#L29

Reply:
Agree. https://github.com/cookpad/tokite/pull/54/commits/d6f71c2c9f3506b3de34146d2ed20406f3987a7a